### PR TITLE
ModelManager integration into ngraph_creator and ops implementations

### PR DESCRIPTION
**Pull request by akodanka**
Refactor ngraph_creator for ModelManager Integration.
Add support for Burst Execution, required for vts tests.
Implementation of operations using the new structure.

### CODE REVIEW

**Review by JaikrishnaNemallapudi — APPROVED** [review]
lgtm

**Review by JaikrishnaNemallapudi — APPROVED** [review]
LGTM

**Review by JaikrishnaNemallapudi — APPROVED** [review]
LGTM

**Review by JaikrishnaNemallapudi — APPROVED** [review]
LGTM

**Review by JeevakaPrabu — COMMENTED** [review]
LGTM

File: ngraph_creator/operations/src/OperationsBase.cpp [hunk 1]
```diff
@@ -23,6 +23,16 @@ std::shared_ptr<ngraph::Node> OperationsBase::transpose(ConversionType type,
     return std::make_shared<ngraph::opset3::Transpose>(input, order_node);
 }
 
+std::shared_ptr<ngraph::Node> OperationsBase::toNCHW(size_t inputIndex,size_t outputIndex) {
+    auto inNode = mNgraphNodes->getOperationOutput(inputIndex).get_node_shared_ptr();
+    if(mNgraphNodes->isForcedNchw(inputIndex))
```
Comment (JeevakaPrabu) [inline ngraph_creator/operations/src/OperationsBase.cpp]: introduce space between if and (

File: ngraph_creator/src/NgraphNodes.cpp [hunk 2]
```diff
@@ -22,6 +24,12 @@ void NgraphNodes::setOperationOutput(size_t index, ngraph::Output<ngraph::Node>
 ngraph::Output<ngraph::Node> NgraphNodes::getOperationOutput(size_t index) {
     return mOperationOutputs[index];
 }
+bool NgraphNodes::isForcedNchw(size_t index) {
```
Comment (JeevakaPrabu) [inline ngraph_creator/src/NgraphNodes.cpp]: Validate the index before using with the vector

File: ngraph_creator/src/NgraphNodes.cpp [hunk 3]
```diff
@@ -22,6 +24,12 @@ void NgraphNodes::setOperationOutput(size_t index, ngraph::Output<ngraph::Node>
...
+    return mForcedNchw[index];
+}
+void NgraphNodes::setForcedNchw(size_t index, bool flag) {
+    mForcedNchw[index] = flag;
```
Comment (JeevakaPrabu) [inline ngraph_creator/src/NgraphNodes.cpp]: same comment as above

**Review by JeevakaPrabu — CHANGES_REQUESTED** [review]

**Review by rairatne — APPROVED** [review]
LGTM

**Review by rairatne — APPROVED** [review]
LGTM

**Review by rairatne — APPROVED** [review]
LGTM

**Review by rairatne — APPROVED** [review]
LGTM

**Review by lakshmishreec — APPROVED** [review]
LGTM

**Review by RavirajSitaram — APPROVED** [review]

File: IENetwork.cpp [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+
+namespace android {
+namespace hardware {
+namespace neuralnetworks {
+namespace nnhal {
+
+bool IENetwork::loadNetwork() {
+    ALOGD("%s", __func__);
+
+    InferenceEngine::Core ie;
```
Comment (JaikrishnaNemallapudi) [inline IENetwork.cpp:18]: Pass plugin.xml path
InferenceEngine::Core ie(std::string("/vendor/etc/openvino/plugins.xml"));

**Review by JaikrishnaNemallapudi — COMMENTED** [review]
Please add plugin.xml path

**Pull request merged**